packages/api/src/controllers/user.ts

```diff
@@ -574,7 +578,7 @@ app.post("/verify-email", validatePost("verify-email"), async (req, res) => {
       });
     }
   }
-  res.status(200).json(cleanUserFields(user));
+  res.status(200).json({});
```

Comment:
I think this one might be needed in some places in the end [1], I think we should update `cleanIUserFields` to remove the PII fields instead. Possibly only for JWT or CORS API key requests, which go to a frontend (backends should be able to fetch it). WDYT?

[1] check this on that [user.ts file](https://github.com/livepeer/livepeer-com/blob/8891a083255bd13ffa4f4b10422e71a267da2231/packages/www/hooks/use-api/endpoints/user.ts) (impl & usage) I mentioned in the other comment

Reply:
Hiding fields makes me nervous, it feels like there's a lot of scope for things to go wrong as names change, different endpoints need to access different bits etc.

We can discuss in the other thread about what's needed for the "customer" creation but are you saying something is depending on the email verification trigger endpoint returning a user object? It wasn't clear from the linked file

Reply:
Yeah agree, I'm [guilty as hell](https://github.com/livepeer/livepeer-com/blob/6c8f97eb3f70ec632f4b6c0624dcdc06412b0822/packages/api/src/controllers/asset.ts#L552) but also do not enjoy the practice. I think on this API it would be better than my mess though, since we know exactly what we are returning and make the clean-up knowing exactly the schema being cleaned-up (thx JS).

> are you saying something is depending on the email verification trigger endpoint returning a user object?

To be honest I don't know! I sent the file just so you could check what response types were needed or not. I wasn't sure if you had found that file since there were no changes to it, but maybe that is really the case and it's a simpler fix for us 😌

Reply:
To be clear, if we are not using those response bodies in our frontend, let's just remove them like you are doing! Less cruft here and attack vectors in the future.